### ISSUE TITLE
suggestion: for Office 15 and its libraries its needs a tiny change  

### ISSUE THREAD

**Issue by andrematzke** [DanwareCreations/VbaModuleManager#2]
Line 26 

25: 'Import all VB code files from the given directory if any)
26:  Dim imports As New **Scripting**.Dictionary

If this is not correct, please feel free to mention it.

BR, AM

**Comment by Rabadash8820** [comment]
Hi @andrematzke, thanks for the suggestion. I'm not sure this is necessary though. If, by Office 15, you mean Excel 2013, then the ModuleManager code should work without fully-qualified type names. Can you verify that you've included all the necessary references, per the [README](https://github.com/Rabadash8820/VbaModuleManager#setup)? If you're getting an error without `Scripting.`, can you please share that error and the line on which it's occurring?

**Comment by andrematzke** [comment]
![2022-03-07 08_07_57-Verweise - RGSM](https://user-images.githubusercontent.com/33298027/156984540-8b009a14-1d8e-489e-aad0-cb319daeb8e0.png)

Yes, ist office 2013.

I've checked, and I think, all the references mentioned in the Readme are there. As you can see some other, too, which are not mentioned, but for this project necessary. Maybe in one of the others there is also a "Dictionary" Type and the compiler needs specification. 

Its when you try to import Modules (Public Sub ImportModules) and come to that line 26
The Messagebox says : Compile Error: illegal use of the keyword "new".

![2022-03-07 08_17_03-Greenshot Editor](https://user-images.githubusercontent.com/33298027/156985815-0ef4bab2-7c4c-48a3-b6d9-65a484c174bd.png)

**Comment by Rabadash8820** [comment]
Ah, so it looks like you are trying to manage modules in a Word document. I have never actually tried to use this code in Word, so I can make no guarantees that it will work. It looks like you've already had to address a couple issues in your own code (e.g., replacing `ThisWorkbook` with `ThisDocument`). So technically, you are using this project in an unsupported way. 😬 However, I _would_ like ModuleManager to be usable in all Office applications (or at least the big ones like Word, Excel, and PowerPoint), so I'm going to leave this Issue open for now until I can refactor `ModuleManager.bas` to be more general (i.e., not have hard-coded references to `ThisWorkbook`).

Thanks for giving me a real-world example of this Word use case!

**Comment by Rabadash8820** [comment]
Okay @andrematzke, I've updated `ModuleManager.bas` to support both Word and Excel. Read back through the setup instructions in the README, as there is now an additional step to configure in which app the manager is running. Please let me know if that fixes your issue!

**Comment by andrematzke** [comment]
Hey @Rabadash8820 , 
1st of all - sorry that I forgot to mention, that I adapted the ModuleManager a little bit to fit to a word document.

I've imported your new ModuleManager.bas into documents of both kinds (Excel & Words) w/o any adaptation (except the const - setting) and replaced there the "adapted" ModuleManager-Version from before.

And it works like a charm. 

I am using this ModuleManager since ~ 2 Weeks, but it already helped me a lot working with the VBA Code !!! Great feature.

BR,
André
